primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.css

```diff
@@ -162,6 +162,10 @@
     vertical-align: middle;
 }
 
+.ui-selectoneradio.ui-grid-responsive label {
+  display: inline;
+}
+
```

Comment:
Not sure if we should do flexing instead. Maybe something to look into in version 14.

Reply:
Yep but its definitely broken now for people migration from the old layout?